gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/iceberg/IcebergTable.java

```diff
@@ -198,7 +198,8 @@ protected DatasetDescriptor getDatasetDescriptor(FileSystem fs) {
    * @param dstMetadata is null if destination {@link IcebergTable} is absent, in which case registration is skipped */
   protected void registerIcebergTable(TableMetadata srcMetadata, TableMetadata dstMetadata) {
     if (dstMetadata != null) {
```

Comment:
are we assuming that `dstMetadata` could never be null ?

Reply:
ok i guess your method javadoc explains

Reply:
yeah, that's the assumption. We would want to rethink later, how we want to support in cases where destination iceberg table is absent.